docs/setup/upgrade/resolving-migration-failures.asciidoc

```diff
@@ -171,7 +171,7 @@ Upgrade migrations fail because routing allocation is disabled or restricted (`c
 
 [source,sh]
 --------------------------------------------
-Unable to complete saved object migrations for the [.kibana] index: [unsupported_cluster_routing_allocation] The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue. To proceed, please remove the cluster routing allocation settings with PUT /_cluster/settings {"transient": {"cluster.routing.allocation.enable": null}, "persistent": {"cluster.routing.allocation.enable": null}}
+Unable to complete saved object migrations for the [.kibana] index: [incompatible_cluster_routing_allocation] The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue. To proceed, please remove the cluster routing allocation settings with PUT /_cluster/settings {"transient": {"cluster.routing.allocation.enable": null}, "persistent": {"cluster.routing.allocation.enable": null}}
```

Comment:
`incompatible_cluster_routing_allocation` is a much better description than `unsupported_cluster_routing_allocation`. In hindsight, `unsupported` wasn't the correct term to use here and is ambiguous (leading to questions like: Is it Kibana or ES that doesn't support the setting?). Naming is hard though 😃